eventuate-local-java-test-util/src/main/java/io/eventuate/local/testutil/CustomDBCreator.java

```diff
@@ -23,17 +28,34 @@ public CustomDBCreator(String dataFile, String dataSourceURL, String driverClass
     this.rootUserPassword = rootUserPassword;
   }
 
-  public void create() {
-    DataSource dataSource = DataSourceBuilder
+  public void create(Optional<SqlScriptEditor> editor) {
```

Comment:
Why not have three overloaded create() methods: public create(), create(SqlScriptEditor editor) and a private one that takes the option.